elixir/apps/web/lib/web/live/sign_up.ex

```diff
@@ -358,7 +359,8 @@ defmodule Web.SignUp do
 
           {:noreply, assign(socket, form: to_form(changeset))}
 
-        {:error, :send_email, _reason, _effects_so_far} ->
+        {:error, :send_email, reason, _effects_so_far} ->
+          Logger.info("failed to send email", reason: inspect(reason))
```

Comment:
Thanks for adding this - we are actually in the process of tuning our telemetry, so this is good timing.

I *think* in most cases this will fail based on the rate limit. Based on our current approach to application monitoring, I believe this might make the most sense as a `warning`.

```suggestion
          Logger.warning("Failed to send sign_up email", reason: inspect(reason))
```